qemu/tests/numa_consistency.py

```diff
@@ -45,9 +45,17 @@ def get_vcpu_used_node(numa_node_info, vcpu_thread):
     params['vcpu_cores'] = 1
     params['vcpu_threads'] = 1
     params['vcpu_sockets'] = params['smp']
+    params['vcpu_maxcpus'] = params['smp']
     params['guest_numa_nodes'] = ""
+    params['mem_devs'] = ""
+    params['backend_mem'] = "memory-backend-ram"
+    params['use_mem'] = "no"
     for node_id in range(len(node_list)):
         params['guest_numa_nodes'] += " node%d" % node_id
+        params['mem_devs'] += "mem%d " % node_id
+        params['numa_memdev_node%d' % node_id] = "mem-mem%d" % node_id
+        params['size_mem%d' % node_id] = "%dM" % \
+            (int(params['mem']) / len(node_list))
```

Comment:
May encounter problems that can't be divisible, cause total memory for all numa nodes don't equal to `-m`.
How about assign a fixed mem size(such as 512M) for each node, and then set `-m` ?

Considering the guest node number are same with host node. The size of each node in host is almost impossible to be smaller than 1G. Very low chance to hit oom error,  how do you think about this ?

Reply:
Good idea. I set 1G for each node, pls help review, thanks.